src/metabase/models/params/chain_filter.clj

```diff
@@ -505,38 +505,48 @@
    options           :- (s/maybe Options)]
   (unremapped-chain-filter remapped-field-id constraints (assoc options :original-field-id original-field-id)))
 
-(defmethod hformat/fn-handler (u/qualified-name ::parens) [_ x]
-  (str "(" (hformat/to-sql x) ")"))
-
-(defn- parens [x]
-  (hsql/call (u/qualified-name ::parens) x))
+(defn- format-parens [_fn [x]]
+  (let [[sql & args] (sql/format-expr x)]
+    (into [(str "(" sql ")")] args)))
```

Comment:
Per Sean's comment on [honeysql#451](https://github.com/seancorfield/honeysql/issues/451#issuecomment-1374067116), couldn't this be something like
```suggestion
  [:nest x])
```
?

([Nest docs](https://github.com/seancorfield/honeysql/blob/v2/doc/special-syntax.md#nest))

Reply:
or maybe skip `format-parens` entirely and put a careful `:nest` into `format-union`?